rudder-agent/SPECS/rudder-agent.spec

```diff
@@ -47,6 +47,9 @@
 # Same goes for the use of the local PCRE install vs. a bundled one
 %define use_system_pcre true
 
+# Should we embeb a uuidgen equivalent (i.e. for AIX)
```

Comment:
And this is the wrong way round - it should say "Should we rely on system uuidgen (if false, provide our own equivalent, i.e. for AIX)"